### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cbertilsson**
Starting this section, this is what I needed to add to my env right now. Anything more you think we should specify here now already?

@wassertim @SadatAnwar

### CODE REVIEW

**Comment by SadatAnwar** [comment]
Hhaha I was about to get to this some day! Thanks for this BTW :)

File: README.md [hunk 1]
```diff
@@ -6,3 +6,14 @@ The lunch meetup app.
...
 The wire-frames for the UI can be accessed/and or worked upon here [wireframe](https://wireframe.cc/dQkepa). 
+
+### Dev setup
+* DB
+```bash
+Role "app" with password "root" on database lunch_mator
+```
+* typings
+```bash
+npm i -g typings
```
Comment (SadatAnwar) [inline README.md]: I think we also need to do an `npm install` ? @wassertim you are the expert

Comment (cbertilsson) [inline README.md]: The "i" there is an npm alias for install @SadatAnwar, but I changed it now to make it clearer.

Comment (SadatAnwar) [inline README.md]: ah! See, I know I am a complete `noob` to all of this front-end fancy schmancy

**Pull request merged**